README.md

```diff
@@ -554,6 +554,7 @@ See also [Are we (I)DE yet?](https://areweideyet.com/) and [Rust Tools](https://
 
 ### Profiling
 
+* [bheisler/criterion.rs](https://github.com/bheisler/criterion.rs) - Statistics-driven benchmarking library for Rust [![Build Status](https://travis-ci.org/bheisler/criterion.rs.svg?branch=master)](https://travis-ci.org/bheisler/criterion.rs)
```

Comment:
`https://travis-ci.org/bheisler/criterion.rs.svg?branch=master` returns a 301. Please can you update it to `https://api.travis-ci.org/bheisler/criterion.rs.svg?branch=master` instead which is where it redirects to.

Reply:
Thanks for noticing!
I fixed the URL.